ubuntu/14.04/standard/Dockerfile

```diff
@@ -48,10 +48,10 @@ RUN apt-get update \
 RUN apt-get update \
  && apt-get install -y --no-install-recommends openjdk-8-jdk \
  && rm -rf /var/lib/apt/lists/*
-RUN apt-get update \
```

Comment:
The template shouldn't have rendered with these lines missing. Can you run update.sh again?

Reply:
Yes, I get how this is working a little better now.  :)